src/mawpy/workflows/workflow1.py

```diff
@@ -0,0 +1,37 @@
+# importing the modules
+import datetime
+import logging
+
+import multiprocessing
+from mawpy.steps.incremental_clustering import incremental_clustering
+import os
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("input_file", help="the CSV file to read the input from")
+parser.add_argument("output_file", help="the CSV file to write the output to.")
+
+logger = logging.getLogger(__name__)
+
+
+def workflow1(
+    input_file,
+    output_file,
+    spatial_constraint,
+    duration_constraint1,
+    duration_constraint2,
+):
+    incremental_clustering(input_file, output_file, spatial_constraint, duration_constraint1)
+
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+    multiprocessing.freeze_support()
+    st = datetime.datetime.now()
+    workflow1(args.input_file, "output_file.csv", 1.0, 0, 300)
```

Comment:
This could be part of another PR, but rather than using a hard-coded temp file name `output_file.csv` we should use the python temp file APIs. This will avoid conflicts with left over files in case of failed runs and will also allow for mutliple concurrent runs not conflicting with each other. Can you log an issue to address this across all workflows?

Reply:
https://github.com/uw-ssec/MAWpy/issues/34